PCbuild/_freeze_module.vcxproj

```diff
@@ -347,6 +367,18 @@
     <Message Text="Updated files: @(_Updated->'%(Filename)%(Extension)',', ')"
              Condition="'@(_Updated)' != ''" Importance="high" />
   </Target>
+  <Target Name="_RebuildDeepFrozen" AfterTargets="_RebuildFrozen" Condition="$(Configuration) != 'PGUpdate'">
+    <Exec Command='$(PythonForBuild) "$(PySourcePath)Tools\scripts\deepfreeze.py" "%(None.OutFile)" "-m" "%(None.ModName)" -o "%(None.IntFile)"' />
```

Comment:
Do we need a separate DeepIntFile?

Reply:
Probably, yeah. It'll be used for dependency analysis, and I _think_ it's more likely to decide that it's always out of date than to incorrectly assume it's up to date, but it's safest to only have one task creating each file during a build.